app/models/district.rb

```diff
@@ -13,14 +13,14 @@ class District < ActiveRecord::Base
   belongs_to :state, required: true
   has_many :senators, through: :state
   has_and_belongs_to_many :zip_codes
-  has_one :representative, class_name: "Legislator"
+  has_one :representative, class_name: "Legislator", dependent: :destroy
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.